### PR TITLE
Refactor names

### PR DESCRIPTION
**Pull request by motiko**
I've renamed Storage class to prevent confusion with [WebStorage](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API). If you have other suggestions feel free to commit directly.

### CODE REVIEW

File: chrome/options/options.tsx [hunk 1]
```diff
@@ -116,16 +137,34 @@ const Options = () => {
...
+            />
             <div className="row container center-content">
-              <a id="restore_defaults_btn" className="button-primary button" onClick={onRestoreBtnClick}>
+              <a className="button-primary button" onClick={onRestoreBtnClick}>
                 Restore Defaults
-              </a> <a id="save_btn" className="button-primary button" onClick={onSaveBtnClick}>Save</a>
+              </a>
+              <a className="button-primary button" onClick={onSaveBtnClick}>
+                Save
+              </a>
```
Comment (p-mazhnik) [inline chrome/options/options.tsx:167]: Here was a space symbol between buttons, so we need to add some margin to keep empty space.

Снимок экрана 2022-01-23 в 15 01 59

File: chrome/options/options.tsx [hunk 2]
```diff
@@ -1,112 +1,133 @@
-import React, { ChangeEventHandler, useEffect, useState } from 'react'
-import { Engines, ITranslateEngine, tesseractLangs} from '../const'
-import { DictCCEngineOptions } from './dict/options'
-import { GoogleTranslateEngineOptions, GoogleTranslateExtEngineOptions } from './google/options'
-import { Storage } from '../services/storage'
-import { showMessage } from './utils'
+import React, { ChangeEventHandler, useEffect, useState } from "react";
```
Comment (p-mazhnik) [inline chrome/options/options.tsx:1]: I see a lot of formatting changes in this pr, so probably we can add `eslint`/`prettier` to enforce the same code style for developers?

**Review by p-mazhnik — COMMENTED** [review]
`Settings` is good

File: chrome/options/options.tsx (hunk 2 above)
Comment (motiko) [reply chrome/options/options.tsx:1]: Can add both. Prettier for formatting eslint for code style rules.

File: chrome/options/options.tsx (hunk 1 above)
Comment (motiko) [reply chrome/options/options.tsx:167]: Nice catch. Will bring it back as soon as im home.

File: tsconfig.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "noEmit": false,
-    "jsx": "react-jsx"
+    "jsx": "react"
```
Comment (p-mazhnik) [inline tsconfig.json:16]: Why did you change this?

Comment (motiko) [reply tsconfig.json:16]: Had a complaint from my linter (which is probably out of date) and assumed it was a typo.
Didn't realise it was intentional. Now after checking documentation I see it's a React 17 feature.
Feel free to change back. (i think both options work)
